packages/js/product-editor/src/components/variations-table/variation-actions-menus/test/variation-quick-update-menu-item.test.tsx

```diff
@@ -0,0 +1,58 @@
+/**
```

Comment:
Love the tests! ❤️

I wonder if we could go a bit higher level and test:

- That using the fill actually fills the correct slot with a `MenuItem`
- The the callbacks are called (you are doing this for `onClick` already)

For example, currently, somebody could change `getMenuItem` so that it returns `null` and these tests would still pass, when clearly something is very wrong in that case.

Also, somebody could change the implementation to not have the `getMenuItem` function, and these unit tests would have to change, when the `getMenuItem` function is really just a small implementation detail that should be able to be changed/renamed without having to change unit tests.

Reply:
> I wonder if we could go a bit higher level and test:
That using the fill actually fills the correct slot with a MenuItem
The the callbacks are called (you are doing this for onClick already)

Done in commit [b54966b](https://github.com/woocommerce/woocommerce/pull/43441/commits/b54966b28828d255b3c984f9e6500a24c66f2223)